sections/editing.include

```diff
@@ -1681,9 +1677,7 @@
   by the user agent as a guide for creating a keyboard shortcut that activates or focuses the
   element.
 
-  If specified, the value must be an <a>ordered set of unique space-separated tokens</a>
-  that are <a>case-sensitive</a>, each of which must be exactly one Unicode code point in
-  length.
+  If specified, the value must be a single printable character: a string exactly one Unicode code point in length.
```

Comment:
Question: can there be whitespace around the character? Do we need to handle trimming of whitespace when processing the attribute?